mimecast/assets/dashboards/mimecast_ttp_attachment.json

```diff
@@ -0,0 +1,1368 @@
+{
+    "title": "Mimecast - TTP Attachment",
+    "description": "This dashboard provides information about the TTP Attachments generated on Mimecast Email Security.",
+    "widgets": [
+        {
+            "id": 5527090904543100,
+            "definition": {
+                "title": "",
+                "banner_img": "https://getlogovector.com/wp-content/uploads/2020/09/mimecast-logo-vector.png",
+                "show_title": true,
+                "type": "group",
+                "layout_type": "ordered",
+                "widgets": [
+                    {
+                        "id": 911775482073612,
+                        "definition": {
+                            "type": "note",
+                            "content": "Gain better visibility into your organization's security events by monitoring Mimecast Email Security - TTP Attachment logs with this dashboard. Targeted Threat Protection(TTP) against attachment is an advanced service, that protects customers from the growing risk of spear phishing and other targeted attacks using email attachments. \n\nThis dashboard provides a comprehensive view of attachment-related security threats, enabling administrators to monitor and analyze malicious activity. \n\nFor more information, see the [Mimecast Email Security Integration Documentation](https://docs.datadoghq.com/integrations/mimecast).\n\n**Tips**\n- Use the timeframe selector in the top right of the dashboard to change the default timeframe.\n- Clone this dashboard to rearrange, modify and add widgets and visualizations.",
```

Comment:
This sentence doesn't make sense: "Targeted Threat Protection(TTP) against attachment is an advanced service, that protects customers from the growing risk of spear phishing and other targeted attacks using email attachments."

I suggest: "Targeted Threat Protection(TTP) Attachment Protection protects customers from spear phishing attacks that use email attachments."

Reply:
I have replace the content as per your suggestion. Let us know if any further changes required. 👍